chain-storage/src/memory.rs

```diff
@@ -71,3 +75,27 @@ where
         self as &BlockStore<Block = Self::Block>
     }
 }
+
+#[cfg(test)]
+pub mod test {
```

Comment:
nits:
- does not need to be public;
- the conventional name is `tests`.

Reply:
Fixed. BTW we have 42 modules named `test` so we're not following the convention very well :-)